core/themes/seven/css/style.css

```
@@ -889,8 +889,15 @@ div.teaser-checkbox .form-item,
 .form-item input.form-radio,
 .form-item input.form-checkbox {
   vertical-align: middle;
+}
+[dir="ltr"] .form-item input.form-radio,
+[dir="ltr"] .form-item input.form-checkbox {
   margin-right: 5px;
```

Our coding standards require that we add `/* LTR */` for things that have a respective RTL:
```suggestion
  margin-right: 5px; /* LTR */
```